package.json

```diff
@@ -1,36 +1,46 @@
 {
     "name": "oid",
     "version": "1.2.0",
-    "keywords":
-        ["object", "id", "identity", "hash", "hashcode", "objid", "oid"],
-    "description":
-        "Utilities for object identity and hashing",
+    "keywords": [
+        "object",
+        "id",
+        "identity",
+        "hash",
+        "hashcode",
+        "objid",
+        "oid"
+    ],
+    "description": "Utilities for object identity and hashing",
     "homepage": "https://github.com/Medium/oid",
     "repository": {
         "type": "git",
         "url": "https://github.com/Medium/oid.git"
     },
-    "licenses": [ {
-        "type": "Apache 2.0",
-        "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
-    } ],
+    "licenses": [
+        {
+            "type": "Apache 2.0",
+            "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+        }
+    ],
     "author": {
         "name": "Dan Bornstein",
         "email": "danfuzz@milk.com",
         "url": "http://www.milk.com/"
     },
-    "maintainers": [ {
-        "name": "Dan Bornstein",
-        "email": "danfuzz@obvious.com",
-        "web": "http://www.milk.com/"
-    } ],
-
+    "maintainers": [
+        {
+            "name": "Dan Bornstein",
+            "email": "danfuzz@obvious.com",
```

Comment:
rofl

Reply:
Do you know about this dude? Apparently he wrote the Dalvik VM

Reply:
🎶 I forgot these comments were public 🎶

Reply:
👋hi! 👋 FWIW I haven't had an `obvious.com` email address in a long time. You can copy my personal address from the block above here if you feel like it.

Reply:
LOL hi Dan!